bitcoin/src/blockdata/transaction.rs

```diff
@@ -990,6 +995,24 @@ impl Transaction {
     }
 }
 
+impl fmt::Display for Transaction {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result { fmt::LowerHex::fmt(self, f) }
+}
+
+impl fmt::LowerHex for Transaction {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        // TODO: Work out how to do this without allocating a string.
+        write!(f, "{}", self.to_hex())
```

Comment:
I think using in `consensus::serde::DisplayWrapper` should work. Making it public might be worth considering but for now `pub(crate)` should suffice.

Reply:
Flagging that I just saw this again, have not tried it.

Reply:
Now used as suggested.